### DISCUSSION TITLE
Why AGPL-3.0 license?

### DISCUSSION THREAD

**Discussion by kaspar030** [cryspen/libcrux-iot#13] [General]
I noticed that the code in this repo is AGPL-3.0 licensed, whereas lubcrux itself is Apache 2.0/MIT. Is that on purpose?

(I'd assume that example, benchmarking and glue code would be as liberally licensed as possible, with the main libcrux license defining the actual terms.)

**Comment by franziskuskiefer** [comment]
Yes, that's on purpose. The IoT variant, including potentially IoT optimised versions will use AGPLv3 and are open for licensing for commercial use.

**Comment by kaspar030** [comment]
In the Q&A after the [talk on 1AD](https://www.youtube.com/watch?v=0GbAMW53RMQ&list=PLTlqQAKXZec8&index=3) it was hinted that the license might change?
